src/cells.rs

```diff
@@ -470,46 +460,29 @@ pub fn window() -> Window<()> {
     cells.insert(make_key("C2"), Cell::new("= A2 * A3 * A4"));
     data.update_values();
 
-    let cells = MatrixView::new(CellDriver).with_num_visible(5, 20);
+    let clerk = Clerk {
+        empty_cell: Cell::default(),
+    };
+
+    let cells = GridView::new(clerk, CellDriver).with_num_visible(5, 20);
 
     let ui = impl_anon! {
-        #[widget {
-            layout = self.cells;
-        }]
+        #[widget]
+        #[layout(self.cells)]
         struct {
             core: widget_core!(),
             data: CellData = data,
-            #[widget(&self.data)] cells: ScrollBars<MatrixView<CellData, CellDriver>> =
+            #[widget(&self.data)] cells: ScrollBars<GridView<Clerk, CellDriver>> =
                 ScrollBars::new(cells),
         }
         impl Events for Self {
             type Data = ();
 
-            fn steal_event(&mut self, cx: &mut EventCx, _: &(), _: &Id, event: &Event) -> IsUsed {
```

Comment:
This isn't supported any more, as such we cannot intercept the `Enter` key and use it to navigate down/up.

Something else is needed. Considering the hacky nature and that nothing else uses event stealing, this was not the right approach.

Reply:
This is #20.